app/models/restaurant.rb

```diff
@@ -0,0 +1,38 @@
+# == Schema Information
+#
+# Table name: restaurants
+#
+#  id                    :uuid             not null, primary key
+#  name                  :string           not null
+#  rating                :integer          default(0)
+#  accept10bis           :boolean          default(FALSE), not null
+#  max_delivery_time_min :integer
+#  address               :string
+#  latitude              :float
+#  longitude             :float
+#  cuisine               :string
```

Comment:
1. cuisine can be empty?
1. What are the possible values for cuisine?

Reply:
the values can be anything, don't see a reason it can't be empty,
I tested indexing and sorting of null valued cuisines and it considers them as highest order values (so they will always come last in an ascending order)

Reply:
if you don't enforce specific cuisines, you might get to the point you will have `Steak`, `Burger`, `burger`, and their ordering will yield: `Burger, Steak, burger`.
Also, you will have hard time grouping by them. Consider using a set of predefined cuisines.

Reply:
I can get a list of cuisines from Zomato by cityID however it won't represent all the possibilities, 
I can enforce a lower-case condition to the DB and convert to lower-case upon update/create
what do you think?

Reply:
start with a small list of 4-5 cuisines, that will be enough for now.